.github/workflows/test.yml

```diff
@@ -0,0 +1,328 @@
+name: Tests
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+  push:
+    branches:
+      - 'master'
+      - 'develop'
+      - 'release/**'
+  workflow_dispatch:
+
+env:
+  NODE_VERSION: 15.9.0
```

Comment:
I see that our all workflow files use `'15.x'` ([changelog.yml](https://github.com/handsontable/handsontable/blob/feature/issue-8652/.github/workflows/changelog.yml#L27), [linter.yml](https://github.com/handsontable/handsontable/blob/feature/issue-8652/.github/workflows/linter.yml#L15) etc.). WDYT to do the same here?